app/src/main/java/org/oppia/android/app/profile/AdminPinActivityPresenter.kt

```diff
@@ -35,7 +35,6 @@ class AdminPinActivityPresenter @Inject constructor(
 
   /** Binds ViewModel and sets up text and button listeners. */
   fun handleOnCreate() {
-    activity.title = activity.getString(R.string.add_profile_title)
```

Comment:
Ah, it seems this might have been wrong before. Glad to see it getting corrected now.

Reply:
All thanks to @rt4914.